packages/kit-bg/src/apis/BackgroundApiBase.ts

```
@@ -307,7 +307,7 @@ class BackgroundApiBase implements IBackgroundApiBridge {
         EAlignPrimaryAccountMode.AlwaysUsePrimaryAccount
       ) {
         // eslint-disable-next-line no-param-reassign
-        targetOrigin = consts.ONEKEY_ALIGN_PRIMARY_ACCOUNT;
+        targetOrigin = consts.ONEKEY_REQUEST_TO_ALL_CS;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)